heat/core/io.py

```diff
@@ -691,10 +691,16 @@ def load(
 
     if extension in __CSV_EXTENSION:
         return load_csv(path, *args, **kwargs)
-    elif supports_hdf5() and extension in __HDF5_EXTENSIONS:
-        return load_hdf5(path, *args, **kwargs)
-    elif supports_netcdf() and extension in __NETCDF_EXTENSIONS:
-        return load_netcdf(path, *args, **kwargs)
+    elif extension in __HDF5_EXTENSIONS:
+        if supports_hdf5():
+            return load_hdf5(path, *args, **kwargs)
+        else:
+            raise ImportError("hdf5 is required for file extension {}".format(extension))
```

Comment:
i think that these should be either `RunTimeErrors` or `TypeErrors`. the issue that this is talking about is that the package is not installed.

Reply:
I was using `ImportError` because it's an external package that needs to be imported.
I don't know if `TypeError` is good here as the function gets the right argument types. `ValueError` maybe. 
A general `RuntimeError` might be the best here.